spec/tests/ladybug_energy_model_model_spec.rb

```diff
@@ -53,12 +53,14 @@
     expect(extension.schema_validation_errors.empty?).to be true
   end
  
+  #add assertions
   it 'can load and validate example face by face model' do
     file = File.join(File.dirname(__FILE__), '../files/example_face_by_face_model.json')
     model = Ladybug::EnergyModel::Model.read_from_disk(file) 
     expect(model.valid?).to be true
     expect(model.validation_errors.empty?).to be true 
-    model.to_openstudio_model
+    face_model = model.to_openstudio_model
```

Comment:
@tanushree04 i would name this `face_openstudio_model = model.to_openstudio_model` or something to remind yourself that it is an OpenStudio Model rather than a Ladybug Model